schema/config_test.go

```diff
@@ -31,7 +31,13 @@ func TestConfig(t *testing.T) {
 			config: `
 {
     "architecture": "amd64",
-    "os": 123
+    "os": 123,
+    "rootfs": {
```

Comment:
@wking 
> This one (and a few others) is still missing architecture and os.

Excuse me I might not understand your comment. `architecture` and `os` seems have been added to all cases.